### DISCUSSION TITLE
Advanced marker using tailwind and Advanced marker pin element

### DISCUSSION THREAD

**Discussion by Felixmurithi** [visgl/react-google-maps#637] [Q&A]
I need help translating the advanced marker example into tailwind css.  If u have an working example of this guide on how to do that please guide. Other examples of advanced markers are also welcome.

**Comment by usefulthink** [comment]
I think as far as the css is concerned, the advanced marker is just a regular html-element with absolute positioning. Is there anything special about tailwind that is causing issues here?

**Comment by Felixmurithi** [comment]
no issues, but u cant pass props to tailwind as done and I am not very good with css animations so writing the component will take some time. I wanted to any help I can get here
